src/adminEntity.ts

```diff
@@ -65,6 +66,26 @@ abstract class AdminEntity {
 
     return [...widgets, ...manyToManyWidgets]
   }
+
+  validate() {
+    if (this.searchFields) {
```

Comment:
avoid nesting, early return

Reply:
I was going to do early return but then I realised that we're going to add more validation logic to this method that doesn't care about searchFields (checking displayFields are existing fields for example)

Reply:
Then split it into `private validateSearchFields()` :))))